src/main/java/com/crud/tasks/controller/TaskController.java

```diff
@@ -27,8 +26,9 @@
     }
 
     @GetMapping(value = "{taskId}")
-    public TaskDto getTask(@PathVariable Long taskId) {
-        return new TaskDto(1L, "test title", "test_content");
+    public Task getTask(@PathVariable Long taskId) {
+        Optional<Task> optionalTask = service.getTask(taskId);
+        return optionalTask.map(taskMapper::mapToTask).orElse(null);
```

Comment:
Lepiej byloby rzucic tutaj wyjatkiem 404 Not Found
zamiast orElse, uzyj orElseThrow(() -> MyCustomException)